Doc/library/os.rst

```diff
@@ -1112,14 +1112,20 @@ or `the MSDN <https://msdn.microsoft.com/en-us/library/z0kc8e3z.aspx>`_ on Windo
    :func:`~os.pwritev` writes the contents of each object to the file descriptor
    and returns the total number of bytes written.
 
-   If the *flags* argument is provided it will invoke the *pwritev2* system
-   call, which modifies the behavior on a per-call basis based on the value
-   of the *flags* argument. If this argument is ommited, the *pwritev* system
-   call will be called instead.
+   :func:`~os.pwritev` will call *pwritev2* system call if available, which modifies
+   the behavior on a per-call basis based on the value of the *flags* argument. If
+   the *flags* argument is provided and different than zero, and *pwritev2* is not
+   available, it will raise a :exc:`NotImplementedError`.
 
    The flags argument contains a bitwise OR of zero or more of the following
    flags:
```

Comment:
of which flags? write an explicit list here.

Reply:
Done in 47f65b5